UnitTests/TestProductsDisplayService.php

```diff
@@ -32,4 +32,62 @@ public function testDisplayProduct_Malformed()
         $this->expectException(TypeError::class);
         ProductsDisplayService::displayProduct('dave');
     }
+
+    public function testDisplayIndividualProduct_Success() {
+        $individualProductsMock = $this->createMock(ProductEntity::class);
```

Comment:
Your test fails - did you forget to update the Entity you're using?

Reply:
yes...